### ISSUE TITLE
A11y: Use of spaces for table layout

### ISSUE THREAD

**Issue by 73nici** [shopware/platform#7208]
*PHP Version:* 8.2   | *Shopware Version:* 6.6.9.0  | *Affected area / extension:* Platform(Default) 

 ----

*_Actual behaviour:_*

The plain text version contains spaces and tab stops, which are used to organise the ordered products as a table.

Screen readers read out the email sequentially, i.e. each word is read out in sequence as it appears in the text. Plain text does not offer the option of semantically labelling content as a table. Screen readers are therefore unable to establish the link between the table cell and the corresponding table header.

When output via a Braille display, all the superfluous spaces are also output. This makes reading very difficult.

![Image](https://github.com/user-attachments/assets/86f17587-c976-46e5-9e26-d4d2a7698606)

*_Expected behaviour:_*

Avoid the visual representation as a table.

Check the plain text version while you are writing. For example, you can read the text to a colleague and check how well the content is conveyed.

The text could, for example, behave more like a list.
```text
Pos 1, 9783453xxx, Katalog Grundschule 2024/2025, Menge 1, Preis 18,99 €.
```
or
```text
Pos 1
-------------------
9783453xxx
Katalog Grundschule 2024/2025
Menge 1
Preis 18,99 €

### Pos 2
-------------------
...
```

*_How to reproduce:_*

- Enable `ACCESSIBILITY_TWEAKS` Feature-Flag
- Login into the shopware administration backend
- Send a test mail via the shopware mail templates module

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40905**



[created from NEXT-40905, comment 542685]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **Low**

 Thank you for your report. After careful consideration, we've concluded that this issue does not require immediate attention or isn't something we can address right now. We prioritize openness and clear communication with our community, so we want to be transparent. We may not resolve all reported issues, and this one might remain unresolved. To stay effective, we periodically clean our backlog, and this issue could be closed in the future.

 For now, we'll keep this ticket open to invite further feedback from the community in case we've underestimated its impact and to allow us to revisit it later.



[created from NEXT-40905, comment 543231]
